arc/simulation/modal_orchestrator.py

```
@@ -518,3 +522,26 @@ def _estimate_execution_cost(self, scenario_count: int, model: str) -> float:
         cost_per_scenario = (1.0 * pricing["cost_per_1k_in"]) + (0.5 * pricing["cost_per_1k_out"])
         
         return scenario_count * cost_per_scenario
+
+    def _normalize_reliability_score(self, result: dict) -> dict:
+        """Normalize reliability scores from 0-100 scale to 0-1 scale for database storage."""
+        if not isinstance(result, dict):
+            return result
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)